openedx/adg/common/lib/mandrill_client/client.py

```diff
@@ -0,0 +1,56 @@
+"""
+Mandrill Client to send ADG emails
+"""
+import logging
+
+import mandrill
+from django.conf import settings
+
+log = logging.getLogger(__name__)
+
+
+class MandrillClient(object):
+    """
+    Mandrill class to send ADG emails
+    """
+    ADG_APPLICATION_SUBMISSION_CONFIRMATION_TEMPLATE = 'adg-application-submission-confirmation'
+
+    def __init__(self):
+        self.mandrill_client = mandrill.Mandrill(settings.MANDRILL_API_KEY)
+
+    def send_mail(self, template_name, user_email, context, attachments=None, subject=None):
```

Comment:
We can avoid multiple parameters in this method by moving the parameters in object as all the parameters belongs to Email